config/config.json

```diff
@@ -23,6 +23,7 @@
     "logLevel": "trace",
     "rpcPort": 8900,
     "network": {
+      "networkPort": 9000,
```

Comment:
Perhaps just port?

Reply:
I was thinking that it would be more understandable inside the code when we use it if we name it networkPort, but I can change it if you think it is better to have just port

Reply:
I agree that we should call it just a port. Because in code you will write config.network.port instead of config.network.networkPort